Sync_app/konturmarket/konturmarket_class_lib.py

```diff
@@ -123,3 +125,43 @@ def sync_assortment(self) -> bool:
             return km_models.KonturMarketDBGood.save_objects_to_db(list_km_goods=km_goods)
 
         return True
+
+    def create_sales_journal(self, date_: datetime.date) -> bool:
+        """Создание журнала розничных продаж в системе ЕГАИС."""
+        if self.login():
+            url_sales_read: KonturMarketUrl = get_url(UrlType.EGAIS_SALES_JOURNAL_READ)
+            response = self._session.get(
+                url=url_sales_read.url,
+                params={
+                    "date": date_.__str__(),
+                },
+            )
+            # Если получили ответ  журнал не содержит записи
+            if response.ok and not response.json().get("day", []).get("rows", []):
+                journal = km_models.KonturMarketDBGood.get_sales_journal_from_db(date_=date_)
+                journal_json = {
+                    "day": {
+                        "day": date_.__str__(),
+                        "rows": [
+                            {
+                                "rowId": str(count + 1),
+                                "alcCode": journal[count]["alcCode"],
+                                "apCode": journal[count]["apCode"].__str__(),
+                                "volume": journal[count]["volume"].__str__(),
+                                "quantity": int(journal[count]["quantity"]),
+                                "price": int(journal[count]["price"]),
+                                "rowType": "New",
+                            }
+                            for count in range(len(journal))
```

Comment:
Почему ты делаешь это так вместо
```suggestion
                            for count, journal in enumerate(journal)
```